Update generating-fsharp-types-from-edmx.md

GoFightNguyen opened the pull request:
# Correcting the getEDMConnectionString function
## In the step "To find or create the connection string for the Entity Data Model," correcting the indentation of the getEDMConnectionString function. Also in that function, correcting the name of the parameter passed into the new SqlConnection() function.

Commit:
Update generating-fsharp-types-from-edmx.md

In the step "To find or create the connection string for the Entity Data Model," correcting the indentation of the getEDMConnectionString function. Also in that function, correcting the name of the parameter passed into the new SqlConnection() function.

qinezh commented:
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx?branch=pr-en-us-1137](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx?branch=pr-en-us-1137)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C12%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610121339487715-1137%5Cworkflow_report.html).

Commit:
Update generating-fsharp-types-from-edmx.md

In the section "Configuring the type provider," add the closing " on the ResolutionFolder static parameter when initializing the edmx. This fix allows the next line to longer look like it is a string.

In the same section, also correct the indentation of the call to getEDMConnectionString

qinezh commented:
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx?branch=pr-en-us-1137](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx?branch=pr-en-us-1137)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C12%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610121352462196-1137%5Cworkflow_report.html).

svick commented on docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx.md:
```diff
@@ -110,11 +110,11 @@ open System.Data.EntityClient
+  let resourceArray = [| "res://*/" |]
+  let assemblyList = [| System.Reflection.Assembly.GetCallingAssembly() |]
+  let metaData = MetadataWorkspace(resourceArray, assemblyList)
+  new EntityConnection(metaData, dbConnection)
```
The function returns a *connection* not a *connection string*, so shouldn't it be named `getEDMConnection`?

svick commented on docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx.md:
```diff
@@ -133,10 +133,10 @@ In this step, you create and configure the type provider with the EDMX connectio
 
 let edmConnectionString =
-getEDMConnectionString("Data Source=SERVER\instance;Initial Catalog=School;Integrated Security=true;")
+  getEDMConnectionString("Data Source=SERVER\instance;Initial Catalog=School;Integrated Security=true;")
```
Similar to above, this is a connection, so I think the variable should be named `edmConnection`.

Also, `EntityConnection` is disposable, so shouldn't `use` be used here instead of `let`? (And probably for `context` below too.)

jasonsnguyen replied:
I followed the suggestions for the name changes. I'm not familiar enough with `use` to be able to make that decision.

cartermp replied:
Agreed on the use of `use` here, since they are disposable.

jasonsnguyen replied:
Replaced those two `let` with `use`

Commit:
Update generating-fsharp-types-from-edmx.md

Based on a code review, renaming EDMConnectionString functions and variables to EDMConnection

qinezh commented:
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx?branch=pr-en-us-1137](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx?branch=pr-en-us-1137)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C12%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610121631064909-1137%5Cworkflow_report.html).

Commit:
Update generating-fsharp-types-from-edmx.md

Based on code-review suggestions, replaced a few let keywords with use since the variables are instances of IDisposable

qinezh commented:
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx?branch=pr-en-us-1137](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx?branch=pr-en-us-1137)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C12%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610121720519944-1137%5Cworkflow_report.html).

cartermp commented:
LGTM.  Thanks, @jasonsnguyen!
